at_client/README.md

```diff
@@ -1,8 +1,103 @@
-<img src="https://atsign.dev/assets/img/@developersmall.png?sanitize=true">
+<img width=250px src="https://atsign.dev/assets/img/@platform_logo_grey.svg?sanitize=true">
 
-### Now for some internet optimism.
+## Now for some internet optimism.
 
-[![Pub Package](https://img.shields.io/pub/v/at_client)](https://pub.dev/packages/at_client)
+[![pub package](https://img.shields.io/pub/v/at_client_mobile)](https://pub.dev/packages/at_client_mobile) [![pub points](https://badges.bar/at_client_mobile/pub%20points)](https://pub.dev/packages/at_client_mobile/score) [![build status](https://github.com/atsign-foundation/at_client_sdk/actions/workflows/at_client_sdk.yaml/badge.svg?branch=trunk)](https://github.com/atsign-foundation/at_client_sdk/actions/workflows/at_client_sdk.yaml) [![gitHub license](https://img.shields.io/badge/license-BSD3-blue.svg)](./LICENSE)
 
 # at_client
-This SDK provides the essential methods for building an app using the @protocol
\ No newline at end of file
+
+### Introduction
+
+SDK that provides the essential methods for building an app using [The @protocol](https://atsign.com). You may also want to look at [at_client_mobile](https://pub.dev/packages/at_client_mobile).
+
+**at_client** package is written in Dart, supports Flutter and follows the
+@platform's decentralized, edge computing model with the following features: 
+- Cryptographic control of data access through personal data stores
+- No application backend needed
+- End to end encryption where only the data owner has the keys
+- Private and surveillance free connectivity
+
+We call giving people control of access to their data "*flipping the internet*".
+
+## Get Started
+
+Initially to get a basic overview of the SDK, you must read the [atsign docs](https://atsign.dev/docs/overview/).
+
+> To use this package you must be having a basic setup, Follow here to [get started](https://atsign.dev/docs/get-started/setup-your-env/).
+
+Check how to use this package in the [at_client installation tab](https://pub.dev/packages/at_client/install).
+
+## Usage
+
+- Set `AtClientPreferences` to your preferred settings.
+
+```dart
+Directory appSupportDir = await getApplicationSupportDirectory();
+AtClientPreference preferences = AtClientPreference()
+        ..rootDomain = 'root.atsign.org'
+        ..namespace = '.my_namespace'
+        ..hiveStoragePath = appSupportDir.path
+        ..commitLogPath = appSupportDirdir.path
+        ..isLocalStoreRequired = true;
+```
+
+- These `preferences` are used for your application.
+
+```dart
+AtClientManager currentAtSign = await AtClientManager.getInstance().setCurrentAtSign(atSign, AtEnv.appNamespace, preferences);
```

Comment:
Variable name "currentAtSign" is incorrect as the call is returning AtClientManager.